api/discord.js

```diff
@@ -0,0 +1,19 @@
+import { Client, Intents } from 'discord.js';
+
+export class Discord {
+    static instance = null;
+
+    static getReadyInstance = async() => {
+        if (Discord.instance !== null) {
+            return discord;
+        }
+        console.log('getting new instance');
+        
+        const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
```

Comment:
@kajahno we don't have access to the members yet. is it possible to update the bot token with the new permission?

Reply:
I don't know how to do this to be honest. I've made you server admin, so now we have the same permissions. Let's pair and find out!